test/dotnet-new.Tests/New3/GivenThatIWantANewApp.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Microsoft.DotNet.Tools.Test.Utilities;
+using System;
+using System.IO;
+using Xunit;
+using FluentAssertions;
+
+namespace Microsoft.DotNet.New3.Tests
+{
+    public class GivenThatIWantANewApp : TestBase
+    {
+        [Fact]
+        public void When_dotnet_new_is_invoked_mupliple_times_it_should_fail()
```

Comment:
Will do. These are here temporarily (for asserting parity with new), when we move out of the `new3` folder we'll rename these.

/cc @seancpeters